scripts/get_archive

```diff
@@ -39,8 +39,13 @@ while [ ${NBWGET} -gt 0 -a ${NBCHKS} -gt 0 ]; do
 
       [ -z "${PKG_SHA256}" -o "${PKG_SHA256}" = "${CALC_SHA256}" ] && break 2
 
-      build_msg "CLR_WARNING" "WARNING" "Incorrect checksum calculated on downloaded file: got ${CALC_SHA256} wanted ${PKG_SHA256}"
-      NBCHKS=$((NBCHKS - 1))
+      if [ "${CHANGE_HASH}" = "yes" ]; then
+        sed -e "s|PKG_SHA256=.*|PKG_SHA256=\"${CALC_SHA256}\"|" -i "${PKG_DIR}/package.mk"
```

Comment:
Can you change regex to `"s|^PKG_SHA256=.*|PKG_SHA256=\"${CALC_SHA256}\"|"` ? This way variable in comment would not be changed.